e2e/internal/e2eutil/testworkload/nginx_workload.go

```diff
@@ -241,11 +240,11 @@ func (n *Nginx) newNginxService() error {
 	return nil
 }
 
-func (n *Nginx) newPingPod(reachable bool) error {
+func (n *Nginx) newPingPod(ctx context.Context, reachable bool) error {
 	name := fmt.Sprintf("%s-ping-job-%s", n.Name, utilrand.String(5))
-	deadline := int64(PollTimeoutForNginx.Seconds())
 
-	cmd := fmt.Sprintf("wget --timeout 5 %s", n.Name)
+	var deadline int64 = 6
+	cmd := fmt.Sprintf("wget --timeout %d %s", deadline-1, n.Name)
```

Comment:
deadline is always (6-1=5) here, right?

Reply:
> deadline is always (6-1=5) here, right?

Yup, but this way it slides if/when you adjust `deadline`.  It will get optimized down to constants during compilation, no?  I haven't actually checked, but that would be a pretty basic compile-time optimization.